index.html

```diff
@@ -1,42 +1,314 @@
+                <li class="tag tag-color fonts-4">boostrap</li>
+                <li class="tag tag-color fonts-4">Ruby</li>
+              </ul>
+            </div>
+            <button class="button fonts-4 button-container" type="button">See Project</button>
+          </div>
+        </section>
+      </section>
+    </section>
+
+    <section class="about headline">
+      <div class="about-content">
+        <div class="padding-lf">
+          <h2 class="about-head primary-text-h1 fonts-1">About me</h2>
+          <p class="about-text content p-color fonts-4">Hello I’m a software developer! I can help
+            you build a product , feature or website
+            Look through some of my work and
+            experience! If you like what you see and
+            have a project you need coded, don’t
+            hestiate to contact me.</p>
+          <button type="button" class="button">Get My Resume</button>
+        </div>
+      </div>
+
+      <div class="bottom-line"></div>
+
+      <ul class="padding-lf skills">
+        <li class="frames-lfs">
+          <div class="frames-lfs">
+            <span class="diamond my-shape"></span>
+            <h2 class="skill-title fonts-4 p-color">Languages</h2>
+          </div>
+
+          <ul class="items">
+            <li class="items-link fonts-4 p-color">Javascript</li>
+            <li class="items-link">Ruby</li>
+            <li class="items-link">Html</li>
+            <li class="items-link">Css</li>
+          </ul>
+        </li>
+        <li class="frames-lfs">
+          <div class="frames-lfs">
+            <span class="diamond my-shape"></span>
+            <h2 class="skill-title fonts-4 p-color">Frameworks</h2>
+          </div>
+
+          <ul class="items">
+            <li class="items-link">Bootstrap</li>
+            <li class="items-link">Ruby On Rails</li>
+            <li class="items-link">RSPec</li>
+            <li class="items-link">CapyBara</li>
+            <li class="items-link">Selenium</li>
+          </ul>
+        </li>
+        <li class="frames-lfs">
+          <div class="frames-lfs">
+            <span class="diamond my-shape"></span>
+            <h2 class="skill-title fonts-4 p-color">Skills</h2>
+          </div>
+
+          <ul class="items">
+            <li class="items-link">Codekit</li>
+            <li class="items-link">Github</li>
+            <li class="items-link">Codepen</li>
+            <li class="items-link">Gitlab</li>
+            <li class="items-link">Terminal</li>
+          </ul>
+        </li>
+      </ul>
+    </section>
+  <section class="form">
+    <h2 class="inter-font contact-title">
+      I'm always interested in hearing about new projects,
+      so if you'd like to chat please get in touch.
+    </h2>
+    <form action="https://formspree.io/f/mgerpyar" method="post" class="contact-form">
+      <div class="form-control">
+        <label for="name" class="sr-only">Full Name</label>
```

Comment:
![image](https://user-images.githubusercontent.com/43698511/130326367-f85bc7ac-edea-4a97-b6e2-384c035168ad.png)

- [ ] As you can see in the picture. The text and the form are aligned. And the text is aligned to the left. Kindly fix these issues.